src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemDeprecationLabel.xaml

```diff
@@ -0,0 +1,71 @@
+<UserControl x:Class="NuGet.PackageManagement.UI.PackageItemDeprecationLabel"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:NuGet.PackageManagement.UI"
+             xmlns:nuget="clr-namespace:NuGet.PackageManagement.UI"
+             xmlns:sys="clr-namespace:System;assembly=mscorlib"
+             mc:Ignorable="d" 
+             d:DesignHeight="300" d:DesignWidth="300">
+  <UserControl.Resources>
+    <ResourceDictionary>
+      <ResourceDictionary.MergedDictionaries>
+        <nuget:SharedResources />
+      </ResourceDictionary.MergedDictionaries>
+
+      <nuget:DeprecationToDeprecationLabelStateConverter x:Key="DataToStateConverter" />
+      <nuget:FormatedStringPartConverter x:Key="FormatedStringPartConverter" />
```

Comment:
It should be FormattedStringPartConverter <--- The name 'FormatedStringPartConverter' is missing a "t"

Reply:
Fixed.